integration/setup/setup.go

```diff
@@ -123,6 +120,7 @@ func NewBackendOpts() *BackendOpts {
 	return &BackendOpts{
 		CappedCleanupInterval:   time.Duration(0),
 		CappedCleanupPercentage: uint8(20),
+		EnableNewAuth:           true,
```

Comment:
I think we should drop the whole function and make zero value (`nil`) useful. If most tests use new auth, we could change this field to `DisableNewAuth` and flip the value

Reply:
That would simplify things :+1: